en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-the-primary-user-store.md

```diff
@@ -26,32 +24,37 @@ Instead of using the embedded database, you can set up a separate repository and
 <tbody>
 <tr class="odd">
 <td><strong>read_only_ldap</strong></td>
-<td><code>org.wso2.carbon.user.core.ldap.ReadOnlyLDAPUserStoreManager</code></td>
+<td><code>org.wso2.carbon.user.core.ldap.UniqueIDReadOnlyLDAPUserStoreManager</code></td>
 <td><p>Use <code>read_only_ldap</code> to do read-only operations for external LDAP user stores.</p></td>
 </tr>
 <tr class="even">
 <td><strong>read_write_ldap</strong></td>
-<td><code>org.wso2.carbon.user.core.ldap.ReadWriteLDAPUserStoreManager</code></td>
+<td><code>org.wso2.carbon.user.core.ldap.UniqueIDReadWriteLDAPUserStoreManager</code></td>
 <td><p>Use <code>read_write_ldap</code> for external LDAP user stores to do both read and write operations.</p></td>
 </tr>
 <tr class="odd">
 <td><strong>active_directory</strong></td>
-<td><code>org.wso2.carbon.user.core.ldap.ActiveDirectoryUserStoreManager</code></td>
+<td><code>org.wso2.carbon.user.core.ldap.UniqueIDActiveDirectoryUserStoreManager </code></td>
```

Comment:
Shall we use the relevant mappings used in `infer.json`. For instance, for `UniqueIDActiveDirectoryUserStoreManager` class, userstore name as `active_directory_unique_id`

Reply:
fixed https://github.com/wso2/docs-apim/pull/2174/commits/25962a162efd96371db974af0b1339864f591d11